.drone.yml

```diff
@@ -0,0 +1,96 @@
+branches:
+  - dev
+  - master
+  - release-*
+  - testing
+
+__buildenv: &buildenv
+  image: rchain/buildenv:latest-java8
+
+__sbtenv: &sbtenv
+  <<: *buildenv
+  environment:
+    - _JAVA_OPTIONS=-Xms2G -Xmx4G -XX:MaxMetaspaceSize=1G
+  volumes:
+    - /var/cache/rchain-build/.sbt:/root/.sbt
+    - /var/cache/rchain-build/.ivy2:/root/.ivy2
+
+clone:
+  git-clone:
+    image: plugins/git:next
+
+pipeline:
+
+  # pr
+
+  sbt-update:
+    <<: *sbtenv
+    commands:
+      - sbt update
+
+  compile:
+    <<: *sbtenv
+    commands:
+      - sbt rholang/bnfc:generate compile test:compile it:compile
+
+  run-unit-tests:
+    <<: *sbtenv
+    commands:
+      - sbt test it:test
+
+  run-code-analysis:
+    <<: *sbtenv
+    commands:
+      - sbt scalafmtCheck coverage
+      - ./scripts/ci/codecov-upload
+
+  # push
+
+  package:
+    <<: *sbtenv
+    commands:
+      - sbt node/docker:publishLocal node/rpm:packageBin node/debian:packageBin node/universal:packageZipTarball
+    volumes:
+      - /var/cache/rchain-build/.sbt:/root/.sbt
+      - /var/cache/rchain-build/.ivy2:/root/.ivy2
+      - /var/run/docker.sock:/var/run/docker.sock
+    when:
+      event: push
+
+  run-integration-tests:
+    <<: *buildenv
+    commands:
+      - ./scripts/ci/run-p2p-test
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /tmp:/tmp
+    when:
+      event: push
```

Comment:
How long do those take on drone?

I'd be in favor of including them in the PR build, just like the overwhelming majority back in July:
https://discordapp.com/channels/391435113415573504/391435113415573507/466034933249605643